samples/node-headless-ssr-experience-edge/config.js

```diff
@@ -33,10 +35,10 @@ const config = {
    */
   port: process.env.PORT || 3000,
   /*
-  * The default language to use in case the context language cannot be determined (this happens
-  * on initial page load, if the language is not specified in the URL)
-  */
-  defaultLanguage: process.env.DEFAULT_LANGUAGE || serverBundle.defaultLanguage
+   * The default language to use in case the context language cannot be determined (this happens
+   * on initial page load, if the language is not specified in the URL)
+   */
+  defaultLanguage: process.env.DEFAULT_LANGUAGE,
```

Comment:
> Great idea, but not sure about introducing dotenv at this point in the v19 release (more of a new feature) since it affects regression already completed. 

Ah great point.

> We should probably still have a fallback here - 'en' is fine.

Works for me, will implement.